config/ppo/WallJump_curriculum.yaml

```diff
@@ -49,20 +49,88 @@ behaviors:
     time_horizon: 128
     summary_freq: 20000
     threaded: true
-
-curriculum:
-  BigWallJump:
-    measure: progress
-    thresholds: [0.1, 0.3, 0.5]
-    min_lesson_length: 100
-    signal_smoothing: true
-    parameters:
-      big_wall_min_height: [0.0, 4.0, 6.0, 8.0]
-      big_wall_max_height: [4.0, 7.0, 8.0, 8.0]
-  SmallWallJump:
-    measure: progress
-    thresholds: [0.1, 0.3, 0.5]
-    min_lesson_length: 100
-    signal_smoothing: true
-    parameters:
-      small_wall_height: [1.5, 2.0, 2.5, 4.0]
+environment_parameters:
+  big_wall_min_height:
+    curriculum:
+    - Lesson0:
+        completion_criteria:
+          measure: progress
+          behavior: BigWallJump
+          signal_smoothing: true
+          min_lesson_length: 100
+          threshold: 0.1
+        value: 0.0
+    - Lesson1:
+        completion_criteria:
+          measure: progress
+          behavior: BigWallJump
+          signal_smoothing: true
+          min_lesson_length: 100
+          threshold: 0.3
+        value: 4.0
+    - Lesson2:
+        completion_criteria:
+          measure: progress
+          behavior: BigWallJump
+          signal_smoothing: true
+          min_lesson_length: 100
+          threshold: 0.5
+        value: 6.0
+    - Lesson3:
+        value: 8.0
+  big_wall_max_height:
```

Comment:
Shouldn't we use a sampler for big_wall_height and modify the WallJump example environment accordingly?

Reply:
strange I thought I changed that.

Reply:
I pushed the new curriculum file for walljump